src/docs/truffle/getting-started/using-the-truffle-dashboard.md

```diff
@@ -0,0 +1,104 @@
+---
+title: Using the Truffle Dashboard
+layout: docs.hbs
+---
+# Using the Truffle Dashboard
+
+When deploying your smart contracts you need to specify an Ethereum account that has enough funds to cover the transaction fees of the deployment. A popular method of doing this is copy-pasting your mnemonic phrase to a gitignored `.env` file so that it can be used for e.g. the `@truffle/hdwallet-provider`. But in general it is a bad practice to copy-paste your keys, especially since we have wallets like MetaMask that can send transactions for us.
```

Comment:
This is super nitpicky and apparently controversial in the grammar community, but I suggest adding a comma here.

```suggestion
When deploying your smart contracts you need to specify an Ethereum account that has enough funds to cover the transaction fees of the deployment. A popular method of doing this is copy-pasting your mnemonic phrase to a gitignored `.env` file so that it can be used for, e.g. the `@truffle/hdwallet-provider`. But in general it is a bad practice to copy-paste your keys, especially since we have wallets like MetaMask that can send transactions for us.
```

Reply:
isn't the rule to put `e.g.` between commas? like "used for, e.g., the @truffle/hdwallet-provider package"

Reply:
Github doesn't let me suggest a change to a suggestion, so I added a separate on to this line.